vit-servicing-station-tests/src/common/clients/rest.rs

```diff
@@ -144,11 +145,33 @@ impl RestClient {
         serde_json::from_str(&content).map_err(RestError::CannotDeserialize)
     }
 
+    pub fn proposals_by_voteplan_id_and_index(
+        &self,
+        request: ProposalsByVoteplanIdAndIndex,
```

Comment:
maybe better to `request` by reference: `request: &ProposalsByVoteplanIdAndIndex`